docs/docs/workflows/upgrade.md

```diff
@@ -6,6 +6,9 @@ You configure the desired versions in your local Constellation configuration and
 To learn about available versions you use the `upgrade check` command.
 Which versions are available depends on the CLI version you are using.
 
+Currently, Azure and GCP are supported for full upgrades.
+Please open an [issue](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md&title=) or [contact us](https://github.com/edgelesssys/constellation#support) directly if you are looking for AWS or OpenStack support.
```

Comment:
```suggestion
:::caution

Upgrades arent't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!

:::
```
For consistency, I'd keep this in line with the existing hints on missing features.
We shouldn't mention OpenStack until we have real support for it.
Is it okay to just speak of "upgrades" instead of "full upgrades" or is a subset of it working on AWS?

Reply:
Measurements can be changed. That's why I included "full". But I wasn't really happy with that either. I think just "upgrades" is fine since this page does not talk about changing measurements anyways.